schemas/apps.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://raw.githubusercontent.com/aiidalab/aiidalab-registry/v1/schemas/apps.schema.json",
+    "$ref": "#/definitions/Welcome",
+    "definitions": {
+        "Welcome": {
+            "type": "object",
+            "propertyNames": {
+              "pattern": "^[A-Za-z][A-Za-z0-9-_]*$"
+            },
+            "additionalProperties": false,
+            "patternProperties": {
+                ".*": {"$ref": "#/definitions/App"}
+            },
+            "title": "Schema for the AiiDAlab apps registry."
+        },
+        "App": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "git_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".git"
+                    ]
+                },
+                "meta_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".json"
+                    ]
+                },
+                "categories": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
```

Comment:
I believe you can make this an Enum, but I am not sure whether that's part of the general JSON Schema or just OpenAPI.

Reply:
Are the possible categories well-defined? I tried to set up the schema such that it best reflects what is currently standardized.

Reply:
Yes, there is a complete file describing them (see [here](https://github.com/aiidalab/aiidalab-registry/blob/master/categories.json)). However, I don't know if you consider that definition enough. I think it is, since they're all simply Enum string values and [`categories.json`](https://github.com/aiidalab/aiidalab-registry/blob/master/categories.json) contains each value's metadata, essentially. Which is more than half of what a schema does ;) (metadata definitions.)

Reply:
It was not clear to me that the "categories.json" file essentially defines the possible categories. If you can confirm that, I will change it to an enum.

Reply:
Indeed it does. That's the point of it. It is also referenced in the README. The categories are not defined anywhere else. Please correct me @yakutovicha if I'm wrong about this, but I don't see why this should not be so.

Reply:
You're 100% right.

Reply:
So I resolved this conversation, but maybe unintentionally, since I thought it ended on the conclusion that a `categories` Enum should be created, or similar, if possible.